src/strategies/digitalax-deco-to-mona/index.ts

```diff
@@ -0,0 +1,62 @@
+import { subgraphRequest } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'onigiri-x';
+export const version = '0.1.0';
+
+const QUICKSWAP_SUBGRAPH =
+  'https://api.thegraph.com/subgraphs/name/sameepsi/quickswap05';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  // Set up the GraphQL parameters and necessary variables
+
+  const decoEthPair = "0x7ecb3be21714114d912469810aedd34e6fc27736";
+  const monaEthPair = "0x3203bf44d434452b4605c7657c51bfeaf2a0847c";
+
+  const holderParams = {
+    pair: {
+      __args: {
+        id: decoEthPair
+      },
+      token1Price: true
+    }
+  };
+
+  // Mona to eth
+  const holderParams2 = {
+    pair: {
+      __args: {
+        id: monaEthPair
+      },
+      token0Price: true
+    }
+  };
```

Comment:
Hi @onigiri-x maybe these prices could be changing in the future or during the voting period? Voting power of users could be changing with time? is there any way to query this with time/block? 

Also i think we can replace these with single query `pairs`
```
{
  pairs(where:{id_in:["0x7ecb3be21714114d912469810aedd34e6fc27736","0x3203bf44d434452b4605c7657c51bfeaf2a0847c"] 

  }){
    token1Price
    token0Price
  }
}
```